Blank Preferance window on Darktable nightly build 20250913 for osx.

### Is there an existing issue for this?

- [x] I checked and did not find my issue in the already reported ones

### Describe the bug

Preferance window opens but is blank !
on my 15.6.1 osx intel

### Steps to reproduce

attempted to edit preferance window.

### Expected behavior

_No response_

### Logfile | Screenshot | Screencast

_No response_

### Commit

_No response_

### Where did you obtain darktable from?

darktable.org / GitHub release

### darktable version

 Darktable nightly build 20250913

### What OS are you using?

Mac

### What is the version of your OS?

15.6.1 osx intel

### Describe your system

_No response_

### Are you using OpenCL GPU in darktable?

None

### If yes, what is the GPU card and driver?

_No response_

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

_No response_

I tried to build in Brew failed.
I was hopeful by the Build.txt for MacPort.
That too failed at cmake command.
here is the error message:

-- Building LibRaw from intree copy
CMake Error at /usr/local/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:227 (message):
  Could NOT find LensFun (missing: LensFun_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/local/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:591 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindLensFun.cmake:35 (find_package_handle_standard_args)
  src/CMakeLists.txt:426 (find_package)

--- a/src/gui/gtk.c
+++ b/src/gui/gtk.c
@@ -4530,7 +4530,7 @@ void dt_gui_commit_on_focus_loss(GtkCellRenderer *renderer, GtkCellEditable **ac
   g_signal_connect(renderer, "editing-started", G_CALLBACK(_commit_on_focus_loss_callback), (gpointer)active_editable);
 }
 
-static void _resize_dialog(GtkWidget *widget, GdkEvent *event, const char *conf)
+static gboolean _resize_dialog(GtkWidget *widget, GdkEvent *event, const char *conf)
 {
   char buf[256];
   GtkAllocation allocation;
@@ -4540,6 +4540,7 @@ static void _resize_dialog(GtkWidget *widget, GdkEvent *event, const char *conf)
   dt_conf_set_int(dt_buf_printf(buf, "ui_last/%s_dialog_height", conf), allocation.height);
   dt_conf_set_int(dt_buf_printf(buf, "ui_last/%s_dialog_x", conf), allocation.x);
   dt_conf_set_int(dt_buf_printf(buf, "ui_last/%s_dialog_y", conf), allocation.y);
+  return FALSE;
 }
 
 void dt_gui_dialog_restore_size(GtkDialog *dialog, const char *conf)

